storage/http_client.go

```diff
@@ -919,20 +919,110 @@ func (c *httpStorageClient) TestIamPermissions(ctx context.Context, resource str
 
 // HMAC Key methods.
 
-func (c *httpStorageClient) GetHMACKey(ctx context.Context, desc *hmacKeyDesc, opts ...storageOption) (*HMACKey, error) {
-	return nil, errMethodNotSupported
+func (c *httpStorageClient) GetHMACKey(ctx context.Context, desc *hmacKeyDesc, accessID string, opts ...storageOption) (*HMACKey, error) {
+	s := callSettings(c.settings, opts...)
+	svc := raw.NewProjectsHmacKeysService(c.raw)
+	// hmacKeyDesc includes configured HMACKey behavior through the HMACKeyOption interface.
+	call := svc.Get(desc.userProjectID, accessID)
+	// TODO: Are there cases where desc.userProjectID != s.userProject?
```

Comment:
IDK the answer to this personally, but I would say possibly - the `s.userProject` comes from the "requestor pays" feature, so I'm guessing that the project where the Key exists could differ from the project making the API call.